packages/protocol/test/genesis/test_config.json

```diff
@@ -80,10 +80,10 @@
     "SignalServiceProxy": "0x1000777700000000000000000000000000000007"
   },
   "param1559": {
-    "yscale": "358298803609133338137582400989",
-    "xscale": "1488514844",
-    "gasIssuedPerSecond": "12500000",
-    "gasExcess": "45450000000"
+    "yscale": "358298803609133338137582400989", // Q: do we need to change this?
```

Comment:
`xscale`, `yscale`, and `gasExcess` are all calculated in _TaikoL2_'s `init()` function, I don't think we need to provide that value here, do we?